test/unit/test_model_flavor.py

```diff
@@ -21,8 +21,8 @@
 logger = logging.getLogger(__name__)
 
 
+@pytest.mark.skipif(platform.system() == "Darwin", reason="Segfaults on Mac")
```

Comment:
Maybe we can change this to `platform.processor() == 'arm'`?

Reply:
The check was there previously on `Darwin', so it did not work for someone else on Mac as stated in the reason. `arm` would turn them of for Linux as well.